tests/integration/explore_user_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package integration
+
+import (
+	"net/http"
+	"testing"
+
+	"code.gitea.io/gitea/tests"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExploreUser(t *testing.T) {
+	defer tests.PrepareTestEnv(t)()
+
+	req := NewRequest(t, "GET", "/explore/users")
+	resp := MakeRequest(t, req, http.StatusOK)
+	assert.Contains(t, resp.Body.String(), `<a class="active item" href="/explore/users?sort=newest&q=">Newest</a>`)
+
+	req = NewRequest(t, "GET", "/explore/users?sort=oldest")
+	resp = MakeRequest(t, req, http.StatusOK)
+	assert.Contains(t, resp.Body.String(), `<a class="active item" href="/explore/users?sort=oldest&q=">Oldest</a>`)
```

Comment:
Instead of this hacky approach, what about adding a uniquely identifiable attribute that won't change such as `data-sortmode="%s"`?

Reply:
I think the HTML code is clear enough here .... so I don't want to add any more unused "attribute" ....

Reply:
`clear` yes.
But not maintainable: If any of these classes or translations changes, the test will fail.
And I'd like to avoid that.
It also fixes your problem below with the 404.

Reply:
Rewritten by 7b996ca2b412ba80fb2a20866ad9efdbf469512c , no any trick there.

And the 404 page only needs to use MakeRequest, it already asserts.